.pre-commit-config.yaml

```diff
@@ -5,8 +5,10 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
       - id: check-yaml
+        exclude: cards/
```

Comment:
Why is the cards/ directory excluded?

Reply:
Because we use an `!include` directive which is not standard YAML. We couldn't find a way for `check-yaml` to accept this. If you have any suggestions on how to fix this without excluding the whole directory this would be great!

Reply:
Seems like ruamel.yaml does not support !includes unless we add a constructor to it or disable safe mode. ruamel.yaml is the library used in the check-yaml pre-commit hook. Unfortunalty we can not change that file unless we want to diverge from the default. 

You could add a second precommit hook with check-yaml and only include cards with the `files` option  and with the -s option enabled in the `args`

Reply:
Though I agree it would be better to check the files that can be checked.. it becomes more tricky as includes may be added later, so we'd have to keep an administration on that. For that reason, we excluded the whole cards folder for now.